changes/2103.feature.rst

```diff
@@ -0,0 +1,2 @@
+A Cookiecutter filter for escaping XML content has been added.
+A Cookiecutter filter for quoting XML attributes appropriately.
```

Comment:
A few issues here:

* The second line is incomplete.
* Towncrier doesn't format multi-line change notes properly, so they should be split into two files called `2103.feature.1.rst` and `2103.feature.2.rst`.
* The `.bugfix.rst` file still exists.

Reply:
Alternatively, this single feature could be written to encompass both:

```suggestion
Cookiecutter filters for escaping XML content and attributes have been added.
```

Regardless - the `.bugfix` version needs to be deleted as well.